src/components/ProfilePage/Leaderboard/Leaderboard.tsx

```diff
@@ -135,21 +190,33 @@ export const Leaderboard = ({ address }: { address?: string }) => {
               pointerEvents: 'none',
             }}
           >
-            <ChevronLeft
-              onClick={previousPage}
-              sx={{ cursor: 'pointer', pointerEvents: 'auto' }}
-            />
+            <Box sx={{ display: 'flex', alignItems: 'center' }}>
+              <FirstPage
```

Comment:
Would be good to be IconButton type so we can remove the sx https://mui.com/material-ui/api/icon-button/